core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.ts

```diff
@@ -240,11 +240,10 @@ angular.module('oppia').component('learnerDashboardPage', {
             $scope.$apply();
           },
           sort: function(e, ui) {
-            /* eslint-disable quote-props */
             // Making top : 0px to avoid irregular change in position.
             ui.helper.css(
+              /* eslint-disable-next-line quote-props */
```

Comment:
Why do we need this disable at all?

Reply:
OK, maybe add a comment explaining the rationale (similar to what @nishantwrp does for typescript checks)?

Reply:
Added the explanation.